packages/server/src/adapters/node-http/types.ts

```diff
@@ -42,6 +45,7 @@ export type NodeHTTPHandlerOptions<
 export type NodeHTTPCreateContextFnOptions<TRequest, TResponse> = {
   req: TRequest;
   res: TResponse;
+  info: TRPCRequestInfo;
```

Comment:
This might break some third party adapters for other libs, should we change target to `next` instead?

Reply:
Why would it break though? You don't have to destruct this prop?

Reply:
Someone might have used that type as a base of their type, no?

e.g. https://github.com/BlairCurrey/trpc-koa-adapter/blob/master/src/index.ts

Reply:
So you mean if they have used the `info` prop for something else then?

```ts
type SomeContextOpts = NodeHTTPCreateContextFnOptions<TReq, TRes>
  & { info: any }
```

Sure - lets add it to v11

Reply:
I mean mainly that maybe they user creation of `NodeHTTPHandlerOptions` will fail as they don't have the `info`? Maybe I'm being too paranoid? If you think it's fine feel free to do it on `main`

Reply:
No you're right I had to change the type of createContext in the websockets example so this is probably breaking something